docs/contributing.md

```diff
@@ -190,3 +190,12 @@ To update a specific version with the latest changes:
 
 1. Remove the version from `versions.json`.
 1. Run `npm run docusaurus docs:version <version>`.
+
+### New Demo Page Added
```

Comment:
It will not be new forever 😅 
Can you please replace that title with something like `Inspect Documentation Style Changes`? Also to include this section **under** (hierarchically) the `Detox Documentation Website` section

Reply:
Ahh, yes. Good point. 👍🏻 
I like your proposed title and changed it.
Reduced heading size to math the hierarchical structure with an h5